lib/jira_api/jira_template_schemas.yaml

```diff
@@ -0,0 +1,21 @@
+# This file holds configuration information on each template
```

Comment:
Something for a future PR - now that there's two template directories - we should merge them into one 
I'd really like there to be a `templates` directory at the top, then a jira/email subdirectory - each with equivalent schema yaml file and template subdirectories underneath